bzt/utils.py

```diff
@@ -1248,15 +1247,11 @@ def _parse_mirrors(self):
 
     def mirrors(self):
         self.log.debug("Retrieving mirrors from page: %s", self.base_link)
-        downloader = ExceptionalDownloader()
+        downloader = ExceptionalDownloader(self.http_client)
         try:
```

Comment:
I'm a bit opposed to this change, as it looks redundant to create `ExceptionalDownloader` and to allocate temporary file just to get HTML, which is a response body.

I'd like to even get rid of ExceptionalDownloader eventually, but this can only be done once each executor/tool are switched to using HTTPClient.

Reply:
My reason to use ExceptionalDownloader is the feature of disabling downloads globally (#885). For that, we need some central point of checking the flag.

Reply:
Sounds reasonable. Let’s merge it then.